libs/@guardian/source-react-components-development-kitchen/src/footer-with-contents/FooterLinks.tsx

```diff
@@ -64,46 +76,68 @@ export const FooterLinks = ({
 	forceColumns = false,
 }: FooterLinksProps): EmotionJSX.Element => {
 	const useColumns = links.length > 3 || forceColumns;
+	const isFooterLink = (
+		link: FooterLink | FooterButton,
+	): link is FooterLink => {
+		return (link as FooterLink).href !== undefined;
+	};
+	const getLink = (link: FooterLink) => {
```

Comment:
Is it worth defining these as component outside of FooterLinks as:
* They have the shape of components e.g. `Props => Jsx.Element`
* they aren't dependant on any values from `FooterLinks`
* Avoid these being generated each time `FooterLinks` is instantiated

Reply:
Yeah they don't need to be defined in the scope of `FooterLinks`, moving them out makes it more readable too IMO 👍